privaterelay/locales/en/privaterelay/brands.ftl

```diff
@@ -0,0 +1,171 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
Taken from [Bedrock](https://github.com/mozilla/bedrock/blob/master/l10n/en/brands.ftl)

Question: Should we par the list down to only strings that are used? I pared the list down considerable from Bedrock, but there are still strings that are not used.